{"cert_user_id": "Kogurex@github", "next_comment_id": 2, "comment": {"1604065012_mirrored_ghost_github": [{"comment_id": 1, "body": "Same Thing Here... My partner at my side using Windows 10 connected with no advices. So i, opened website shown and loaded successfully. Here's the terminal description:\n\n`\n- ~/ZeroBundle$ sudo ./ZeroNet.sh \n- Starting ZeroNet...\n- OpenSSL loaded, version: 01000207F\n- Version: 0.3.6 r1089, Python 2.7.11 |Continuum Analytics, Inc.| (default, Dec  6 2015, 18:08:32) \n  [GCC 4.4.7 20120313 (Red Hat 4.4.7-1)], Gevent: 1.0.2\n- Creating FileServer....\n  TorManager Tor controller connect error: error: [Errno 111] Connection refused in TorManager.py line 154 > socket.py line 344\n- Creating UiServer....\n- Removing old SSL certs...\n- Starting servers....\n- Ui.UiServer --------------------------------------\n- Ui.UiServer Web interface: http://127.0.0.1:43110/\n- Ui.UiServer --------------------------------------\n- Opening browser: default_browser...\n- FileServer Checking port 15441 using portchecker.co...\n- FileServer [BAD :(] Port closed: Port 15441 is closed.\n- FileServer Trying to open port using UpnpPunch...\n- FileServer Checking port 15441 using portchecker.co...\n- FileServer [BAD :(] Port closed: Port 15441 is closed.\n- FileServer Checking port 15441 using canyouseeme.org...\n- FileServer [BAD :(] Port closed: Error: I could not see your service on 186.67.115.230 on port (15441) Reason: Connection timed out\n- FileServer Upnp mapping failed :( Please forward port 15441 on your router to your ipaddress\n  `\n", "added": 1459328087, "modified": 1459328491, "source_link": "https://github.com/HelloZeroNet/ZeroNet/issues/223#issuecomment-203395898", "source_type": "github"}]}}